torchgen/api/functionalization.py

```diff
@@ -7,6 +7,7 @@
     boolT,
     ConstRefCType,
     CType,
+    InverseReturnModeT,
```

Comment:
I would probably just define `InverseReturnModeT` directly in this file, instead of `types.py` (it's a functionalization detail, probably not worth cluttering up the generic list of C++ types we have there)